subgraph/subgraph-mainnet.yaml

```
@@ -0,0 +1,28 @@
+specVersion: 1.3.0
+indexerHints:
+  prune: auto
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: PDPVerifier
+    network: filecoin
+    source:
+      address: '0x1790d465d1FABE85b530B116f385091d52a12a3b'
```

```suggestion
      address: ${{ vars.PDP_VERIFIER_ADDRESS }}
```

Ditto, we could define `PDP_VERIFIER_ADDRESS` var for every environment and load it from variables.